.azuredevops/azure-pipelines.yml

```diff
@@ -11,36 +11,35 @@ variables:
   - template: variables.yml
 
 pool:
-  vmImage: "ubuntu-latest"
+  vmImage: ubuntu-latest
 
 steps:
   - task: Hey24sheep.flutter.flutter-install.FlutterInstall@0
     displayName: "Flutter Install"
 
   - task: gitversion/setup@0
-    retryCountOnTaskFailure: 3
     inputs:
-      versionSpec: "5.12.0"
-    displayName: "Install GitVersion"
+      versionSpec: '5.10.1'
```

Comment:
Why downgrade gitversion?

Reply:
that was during test, i'll double check if it works with 5.12

Reply:
updated